devicon.json

```
@@ -4475,6 +4475,35 @@
         "color": "#4051b5",
         "aliases": []
     },
+    {
+        "name": "unrealengine",
+        "tags": [
+            "c++",
+            "engine",
+            "game"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
```

```suggestion
                "original",
                "original-wordmark"
```
The `aliases` system is a bit confusing. Here, you need to put the actual SVG files that are used as the `font` versions. The `aliases` attributes below would then convert the names to its aliases when making the classes.